sdf/composition/overview.mdx

```diff
@@ -91,172 +91,10 @@ The sections are as follows:
 * `functions` - defines functions in the package
 * `dev` - defines substitutions used during development and test
 
-
```

Comment:
Why is "Build and Test a Package" removed?

Reply:
It is repeated in Quickstart

Reply:
We should keep in package and remove in Quickstart.

Reply:
I tried to follow the current docs.

First did https://www.fluvio.io/sdf/composition/overview and then https://www.fluvio.io/sdf/composition/quickstart

When I was in quickstart I got annoyed because it was exactly the same of the last part of overview. Are you suggesting to remove quickstart and only keep all in overview?

Reply:
I've created the Quickstart when we added packages for the first time, so it's due for an update.
The Quickstart should use inline dataflows instead. I consider packages as an advanced feature.

Reply:
But this is the quickstart for composition, not the quickstart for sdf, Should I remove it?

Reply:
I updated my comment above ☝️